### PR TITLE
changedGoToAPIToGoToService

### PR DESCRIPTION
**Pull request by john-paul-keenan**
as api is depreciated, I changed the link to go the newly created services/routes page

**NOTE**: Please read the CONTRIBUTING.md guidelines before submitting your patch,
and ensure you followed them all:

https://github.com/Kong/getkong.org/blob/master/CONTRIBUTING.md

### Summary

SUMMARY_GOES_HERE

### Full changelog

* [Edited docs to ...]
* [Added cool new feature ...]
* ...

### Issues resolved

Fix #XXX

### Checklist:

- [ ] [Commit message & atomicity](https://github.com/Kong/getkong.org/blob/master/CONTRIBUTING.md#commit-atomicity) checked
- [ ] Documentation 
- [ ] Spellchecked my updates
- [ ] Ready to be merged

### CODE REVIEW

File: app/docs/enterprise/0.32-x/getting-started/quickstart.md [hunk 1]
```diff
@@ -76,7 +76,7 @@ $ kong reload
 
 Now that you have Kong EE running you can interact with the Admin API and GUI.
 
-To begin, go to [Adding your API &rsaquo;][adding-your-api]
+To begin, go to [Configuring a Service &rsaquo;][configuring-a-service]
```
Comment (darrenjennings) [inline app/docs/enterprise/0.32-x/getting-started/quickstart.md]: you will also need to change the link at the bottom of this document in order for the link to work.

**Comment by coopr** [comment]
@john-paul-keenan are you able to update this PR as requested and resolve merge conflicts?

**Comment by john-paul-keenan** [comment]
@coopr I actually need someone with a bit more git knowledge to help. I'm not sure how to correct this

**Comment by coopr** [comment]
@john-paul-keenan you might want to just close this PR, and open a new one that fixes this in the EE 0.32 _and_ 0.33 docs at the same time.

**Pull request closed without merging**